implementations/c/provekit-realize-c-core/tests/conformance.py

```
@@ -0,0 +1,294 @@
+    if "error" in response:
+        raise RuntimeError(f"realizer returned error: {response['error']}")
+    result = response.get("result", {})
+    source = result.get("source")
+    if not isinstance(source, str):
+        raise RuntimeError(f"realizer response missing result.source: {response}")
+    return source
+
+
+def compile_source(cc, source_path, out_path, fixture_name):
+    cmd = [cc, "-Wall", "-Wextra", "-Werror", str(source_path), "-o", str(out_path)]
+    proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)